applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -600,33 +567,33 @@ def _is_fm_ale_step(self):
             return False
 
     def __UpdateFMALEStepCounter(self):
-        if self.__fm_ale_is_active:
-            if (self._is_fm_ale_step()):
+        if self._FmAleIsActive():
+            if (self.__IsFmAleStep()):
                 # Reset the FM-ALE steps counter
                 self.fm_ale_step = 1
             else:
                 # Update the FM-ALE steps counter
                 self.fm_ale_step += 1
 
-    def _set_virtual_mesh_values(self):
-        if self._is_fm_ale_step():
+    def __SetVirtualMeshValues(self):
```

Comment:
I know now it is already done but for the future I would split those changes into a separate PR
esp with one as big and delicate as this

Reply:
Well, I thought you wanted me to also tackle the style guide issues. My apologies.